app/controllers/idv/hybrid_mobile/entry_controller.rb

```diff
@@ -0,0 +1,80 @@
+module Idv
+  module HybridMobile
+    # Controller responsible for taking a `document-capture-session` UUID and configuring
+    # the user's Session to work when they're forwarded on to document capture.
+    class EntryController < ApplicationController
+      include HybridMobileConcern
+
+      before_action :track_document_capture_session_id_usage
+      before_action :validate_document_capture_session_id
+      before_action :validate_document_capture_user_id
```

Comment:
Why do these as before actions rather than at the beginning of `#show`?

Reply:
Honestly, I'm not sure. I feel like I keep seeing a preference for `before_action` in this codebase, but with only 1 action it's probably overkill

Reply:
To me, the before actions hide the main purpose for this controller, so it would be more clear to have it in show before redirecting to document_capture.